support-frontend/assets/components/footerCompliant/footer.jsx

```diff
@@ -50,11 +50,30 @@ function Footer({
               {disclaimer && <ContribLegal countryGroupId={countryGroupId} />}
             </Rows>
           </div>
-        </Content>
+        </FooterContent>
       }
-      <Content border appearance={appearance}>
+      <FooterContent border>
+        <ul css={linksList}>
+          <li css={link}>
+            <a href="https://www.theguardian.com/help/privacy-policy">Privacy Policy</a>
```

Comment:
Would the [Link component](https://www.theguardian.design/2a1e5182b/p/43c26b-link/b/048fd1) be of use here?